src/Datastore/DatastoreSessionHandler.php

```diff
@@ -204,9 +216,21 @@ public function read($id)
 
     /**
      * Write the session data to Cloud Datastore.
+     *
+     * @param string $id Identifier used to construct a {@see Key} for the {@see Entity} to be written.
```

Comment:
these need to be fully-qualified to get parsed by the doc generator
<img width="153" alt="screen shot 2017-08-08 at 3 14 25 pm" src="https://user-images.githubusercontent.com/89034/29090019-4c745894-7c4c-11e7-84fb-b875d74ce100.png">